doc/sphinx/source/recipes/recipe_climwip.rst

```diff
@@ -15,22 +15,36 @@ Available recipes and diagnostics
 
 Recipes are stored in esmvaltool/recipes/
 
-    * ``recipe_climwip.yml``
+    * ``recipe_climwip_test_basic.yml``
+    * ``recipe_climwip_test_performance_sigma.yml``
 
-Diagnostics are stored in esmvaltool/diag_scripts/weighting/
+(A recipe adding a 'real' case will be added in due time.)
 
-    * ``climwip.py``: Compute weights for each input dataset
-    * ``weighted_temperature_graph.py``: Show the difference between weighted and non-weighted temperature anomalies.
+Diagnostics are stored in esmvaltool/diag_scripts/weighting/climwip/
+
+    * ``main.py``: Compute weights for each input dataset
+    * ``calibrate_sigmas.py``: Compute the sigma values on the fly
+    * ``core_functions.py``: A collection of core functions used by the scripts
+    * ``io_functions.py``: A collection of input/output functions used by the scripts
+
+Plot scripts are stored in esmvaltool/diag_scripts/weighting/
+
+    * ``weighted_temperature_graph.py``: Show the difference between weighted and non-weighted temperature anomalies as time series.
+    * ``weighted_temperature_map.py``: Show the difference between weighted and non-weighted temperature anomalies as map.
+    * ``plot_utilities.py``: A collection of functions use by the plot scripts.
 
 
 User settings in recipe
 -----------------------
 
-1. Script ``climwip.py``
+1. Script ``main.py``
 
   *Required settings for script*
-    * ``performance_sigma``: shape parameter weights calculation (determined offline). Can be skipped or not set if ``performance_contributions`` is skipped or not set.
-    * ``independence_sigma``: shape parameter for weights calculation (determined offline). Can be skipped or not set if ``independence_contributions`` is skipped or not set.
+    * ``performance_sigma`` xor ``calibrate_performance_sigma``: If ``performance_contributions`` is given exactly one of the two has to be given. Otherwise they can be skipped or not set.
+
+        * ``performance_sigma``: float setting the shape parameter for the performance weights calculation (determined offline).
+        * ``calibrate_performance_sigma``: dictionary setting the performance sigma calibration. Has to contain at least the key-value pair specifying ``target``: ``variable_group``. Optional parameters for adjusting the calibration are not yet implemented. WARNING: It is highly recommended to visually inspect the graphical output of the calibration to check if everything worked as intended. In case the calibration fails, the best performance sigma will still be indicated in the figure (see example figure [_fig_climwip_5] [test](#_fig_climwip_5) below) but not automatically picked - the user can decide to use it anyway by setting it in the recipe (not recommenced).
```

Comment:
```suggestion
        * ``calibrate_performance_sigma``: dictionary setting the performance sigma calibration. Has to contain at least the key-value pair specifying ``target``: ``variable_group``. Optional parameters for adjusting the calibration are not yet implemented. WARNING: It is highly recommended to visually inspect the graphical output of the calibration to check if everything worked as intended. In case the calibration fails, the best performance sigma will still be indicated in the figure (see example :numref:`fig_climwip_5` below) but not automatically picked - the user can decide to use it anyway by setting it in the recipe (not recommenced).
```

this should do it

Reply:
and maybe we can trim down the line length a bit...

Reply:
ah sorry didn't see that before. You mean manual line breaks? Will do.